tests/RuleSet/SampleRulesBad.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\RuleSet;
+
+/**
+ * Sample external RuleSet.
+ *
+ * This class is not extending the required class `\PhpCsFixer\RuleSet\RuleSetDescriptionInterface`,
+ * so it will not be a valid class to be registered as a RuleSet.
+ */
+class SampleRulesBad
+{
+    public const NAME = '@RulesBad';
```

Comment:
When I see a public constant I assume it will be used outside of the class. Let's make it private or remove it as it is only used once in `getName` method.

Reply:
I'll make it private.

Reply:
I've removed it and used literal string in `getName()`.